src/py/mat3ra/made/tools/utils/helpers.py

```diff
@@ -0,0 +1,71 @@
+from typing import Callable, List, Literal
+
+import numpy as np
+from pydantic import BaseModel
+from scipy.integrate import quad
+from scipy.optimize import root_scalar
+
+AXIS_TO_INDEX_MAP = {"x": 0, "y": 1, "z": 2}
+EQUATION_RANGE_COEFFICIENT = 5
+
+
+class FunctionHolder(BaseModel):
+    @staticmethod
+    def get_derivative(*args, **kwargs):
+        raise NotImplementedError
+
+    @staticmethod
+    def get_arc_length_equation(*args, **kwargs):
+        raise NotImplementedError
+
+    @staticmethod
+    def get_transform_coordinates(*args, **kwargs):
```

Comment:
We can define the docstrings in the base class

Reply:
But the args are different for every child